htdocs/core/lib/pdf.lib.php

```diff
@@ -1129,6 +1129,7 @@ function pdf_writeLinkedObjects(&$pdf, $object, $outputlangs, $posx, $posy, $w,
 	$linkedobjects = pdf_getLinkedObjects($object, $outputlangs);
 	if (! empty($linkedobjects))
 	{
+		$posy+=3;
```

Comment:
If you add $posy+=3 outside of loop, it means you move cursor down even if nothing is output (if linkedobjects is empty) ?